projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts

```diff
@@ -96,6 +96,8 @@ export class WorkAllocationService {
    */
   public completeTask(taskId: string): Observable<any> {
     if (!this.isWAEnabled()) {
+      this.alertService.setPreserveAlerts(true);
+      this.alertService.warning({ phrase:'WA Enabled: A task could not be completed successfully. Please complete the task associated with the case manually.'});
```

Comment:
I would change this to "completeTask: Work Allocation is not enabled, so the task could not be completed. Please complete the task associated with the case manually."

Reply:
updated